draft-ietf-wimse-s2s-protocol.md

```diff
@@ -324,6 +324,15 @@ Note that per {{RFC9110}}, header field names are case insensitive;
 thus, `Workload-Identity-Token`, `workload-identity-token`, `WORKLOAD-IDENTITY-TOKEN`,
 etc., are all valid and equivalent header field names. However, case is significant in the header field value.
 
+### Including Additional Claims {#add-claims}
+
+The WIT is a JSON structure and therefore can be trivially extended by adding more claims than are defined in the current specification.
+This, however, could result in interoperability issues, which the following rules are addressing.
+
+* Deployers may freely add claims to the WIT beyond those defined here, in closed environments. Such claims SHOULD be collision-resistant, such as `example.com/myclaim`.
+* A recipient that does not understand such claims MUST ignore them, as per Sec. 4 of {{RFC7519}}.
+* Outside of closed environments, new claims MUST be registered with IANA {{IANA.JWT.CLAIMS}} before they can be used.
```

Comment:
seems like these rules are not aligned with similar content from RFC 7519 itself, which doesn't seem good

Reply:
@bc-pi In what way?